jlh/WeatherApp/app/src/main/java/com/e_conomic/weatherapp/domain/mappers/ForecastDataMapper.kt

```diff
@@ -24,12 +19,12 @@ class ForecastDataMapper {
     }
 
     private fun convertToModelForecast(forecast: Forecast): ModelForecast {
-        val description: String
-        val iconUrl: String
+        val description: String?
+        val iconUrl: String?
 
         if (forecast.weather.isEmpty()) {
-            description = NO_DESCRIPTION
-            iconUrl = NO_ICON
+            description = null
```

Comment:
Yup, of course. Thought something was a it off haha :) I agree, it's much nicer!